manim/scene/scene.py

```diff
@@ -90,10 +91,14 @@ def __init__(
             self.mouse_drag_point = OpenGLPoint()
 
         if renderer is None:
-            self.renderer = CairoRenderer(
-                camera_class=self.camera_class,
-                skip_animations=kwargs.get("skip_animations", False),
-            )
+            skip_animations = kwargs.get("skip_animations", False)
+            # If renderer is unspecified, use the configs for a sensible default.
+            if config["use_opengl_renderer"]:
+                self.renderer = OpenGLRenderer(skip_animations=skip_animations)
```

Comment:
Instead of passing `skip_animations` here the renderer can read it from the scene reference it holds. The `skip_animations` kwarg passed to OpenGLRenderer should be removed at some point.

Reply:
I think this is made moot with the render / scene / camera refactoring from the other comment.